src/intervals/conversion.jl

```diff
@@ -30,15 +24,8 @@ function convert{T<:AbstractFloat, S<:Real}(::Type{Interval{T}}, x::S)
     # use @round_up and @round_down here?
 end
 
-function convert{T<:AbstractFloat}(::Type{Interval{T}}, x::Float64)
-    II = convert(Interval{T}, rationalize(x))
-    # This prevents that rationalize(x) returns a zero when x is very small
-    if x != zero(x) && II == zero(Interval{T})
-        return Interval(parse(T, string(x), RoundDown),
-                        parse(T, string(x), RoundUp))
-    end
-    II
-end
+convert{T<:AbstractFloat}(::Type{Interval{T}}, x::Float64) =
```

Comment:
This is just far too slow for such a basic operation that is used all over the place, e.g. in `Interval(0.1) * 0.3`:

```
julia> @btime Interval(0.1) * 0.3
  700.447 ns (10 allocations: 416 bytes)
Interval(0.03, 0.03000000000000001)

julia> @btime Interval(0.1)
  7.612 ns (0 allocations: 0 bytes)
Interval(0.1, 0.1)

julia> @btime convert(Interval{Float64}, 0.3)
  610.783 ns (10 allocations: 416 bytes)
Interval(0.3, 0.30000000000000004)

julia> @btime Interval(prevfloat(0.3), nextfloat(0.3))
  12.497 ns (0 allocations: 0 bytes)
```
(A factor of 50 difference!)

I suggest we use `parse` for something like `convert(Interval, "0.3")` (when a string is given), and simply use `convert(::Type{Interval{Float64}, x) = Interval(prevfloat(x), nextfloat(x))` for floats.